Core/MIPS/ARM/ArmAsm.cpp

```diff
@@ -70,9 +71,84 @@ namespace MIPSComp {
 
 using namespace ArmJitConstants;
 
-void ArmJit::GenerateFixedCode()
-{
-	enterCode = AlignCode16();
+void ArmJit::GenerateFixedCode() {
+	const u8 *start = GetCodePtr();
+
+	// LR == SCRATCHREG2....  Maybe we could find some other register to use to avoid having to
+	// push LR?
```

Comment:
Yeah, it's no big deal really. I'll probably just remove this comment.

We do only call it when flushed, I think, so you're right that we could probably just grab R2 or whatever, but I probably won't bother for now.